include/rdma/fi_ext.h

```diff
@@ -33,13 +34,31 @@
 #ifndef FI_EXT_H
 #define FI_EXT_H
 
+#include <stdbool.h>
 #include <rdma/fabric.h>
 
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/*
+ * Each provider can define provider-specific values by
+ * choosing an unique 12-bit provider-specific code to
```

Comment:
nit: This is really only 11-bits, since the upper bit must be 1

Reply:
Thanks, fixed in rev2.